educator/src/Dscp/Educator/Web/Server.hs

```diff
@@ -105,7 +105,7 @@ educatorCors = cors $ const $ Just $
     , corsMethods = methodsCoveringAPI @['GET, 'POST, 'PUT, 'DELETE] @EducatorWebAPI
     }
 
-serveEducatorAPIsReal :: EducatorWorkMode ctx m => Bool -> m ()
+serveEducatorAPIsReal :: EducatorWorkMode ctx m => Bool -> m a
```

Comment:
In launcher runner you return `Void`, here you return `forall a. a`.

Reply:
Forgot about runner :+1: